notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.plot(x, q.prob(x), label="q")
</pre><p>I would suggest to use 3 different line style, for example <code>normal</code>, <code>dashed</code>,  <code>dash-dotted</code>. </p><p>Reason why I'm saying is, once Dr. Kevin mentioned that sometimes color blind people read this books and they won't able to distinguish these three lines, but different style can be easily recognized  </p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>